manifests/testing/disks-images-provider.yaml.in

```diff
@@ -114,6 +114,27 @@ spec:
               - /ready
             initialDelaySeconds: 10
             periodSeconds: 5
+        - name: loopdev
+          command:
+          - sh
+          - -c
+          - |
+            while true; do
+              for i in $(seq 100); do
+                if ! [ -e /dev/loop$i ]; then
+                  mknod /dev/loop$i b 7 $i
+                fi
+              done
+              sleep 100000000
```

Comment:
This is a bit more than 1000 days, maybe `sleep 1000d`?
Also, since the task is done, why can't this script just finish?

Reply:
I assume it was done that way because that's what kubernetes/test-infra did. I can't think of a reason to repeat something every ~3 years. removed, thanks.

Reply:
I just realized why it's there (and why the CI was failing):
The default restartPolicy is "Always", so if it exits, it just restarts it again. It ends up looking like a failed container, it's gets to CrashLoopBackOff because it keeps being restarted.
If we want to exit, we need a restartPolicy of "OnFailure", which isn't an option on a DaemonSet.
So I made it into a Pod, where it is possible.

Reply:
Sorry for barging in just now, but couldn't this `loopdev` container be an [initContainer](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#understanding-init-containers) ? I get the feeling they exist for stuff exactly like this.

Reply:
Thanks, I'll give it a look.
BTW, I am putting off this cleanup for a second PR. My previous attempt had some minor issue for the CI and it was taking me a while to pin it down, and I don't want to block people for longer.

Reply:
:+1: this PR is waiting for a long time now.. let's get it in and improve it later.
About the sleep, you can just do `sleep INF` when you the process to never wake up.